### ISSUE TITLE
intro to the RStudio server

### ISSUE THREAD

**Issue by zolli22** [RhoBott/data_at_reed#82]
going off of thoughts expressed in  #76 : 

> navigating the server can be confusing, and if we truly want to have instructions for the most novice user, maybe we should add an "intro to RStudio/the server" section where we outline what the different panels/tabs/windows are in RStudio, what they do, and where they are? and then, additional info about file paths, creating folders, making sure you know where you are, setting a working directory maybe?
> 
> food for thought.

is this something we want to create? what specific things would you like to see explained?

**Comment by RhoBott** [comment]
[ going to let mlabbies weigh in before i do, but _definitely have some thoughts-feelings_ here ]

**Comment by avawillis** [comment]
I like this idea, and it's a good opportunity to establish some base vocabulary (e.g. "console") that we then don't have to re-explain in subsequent documentation pages. For example, we wouldn't have to describe where the files pane is every time we mention it. 

What I would want to include is:

1. Short description of the console, the environment pane, the files pane, etc., as well as any relevant tabs contained therein

2. There's already documentation about RScripts and .Rmds -- maybe we could even relocate that documentation to this new hypothetical page?

3. As an elaboration of R Markdown: creating code chunks, knitting, important chunk options e.g. `eval = FALSE`

4.  A description of the process of creating an R project (I don't think this exists yet, but I may be wrong) 

5. Relating to the above: maybe we could talk about doing GitHub version control? This isn't necessary super pertinent to the "novice" user, but considering there are pages about web scraping and other more advanced topics, I think this could be something useful to mention (also whenever people are confused about GitHub we could direct them to a nice help page for it!) -- although, if we create this, it might be something to put in an entirely separate section

That's all I can think of at the moment that isn't already encompassed by other existing pages!

**Comment by anaqb** [comment]
totally agree with Ava! I think this would be a great thing to have. Also a good thing to link when we do mention any vocabulary that may be more R-specific/easy to forget :)

Things I would like to see:
-seconding Ava on creating a project and also navigating between projects and also within a project -- difference between creating a new project and a new script
- section for importing data into the environment and a breakdown of the panels in the popup that comes up when you go to import data

That is all for now :) Thank you for getting this going, Ingrid

**Comment by joshyam-k** [comment]
Yeah I think this is definitely a page we want to have and it'd be good to keep it short and use a lot of pictures!

I almost wonder if we should have one page dedicated to just the different panels in RStudio UI + how to open or create new files and then create other pages to address some of the other wonderful ideas you all have (flepaths, projects, rmarkdown feature explanations).

I think the whole Rstudio interface can be a lot when youre starting to learn R, so it'd be nice to see one page dedicated to just talking through all thats going on!

**Comment by RhoBott** [comment]
I am a huge fan of expanding our documentation to cover all user needs, and also want to make sure that we keep things sufficiently contained at least _for the moment_.

Ava, to your 
>  it's a good opportunity to establish some base vocabulary (e.g. "console") that we then don't have to re-explain in subsequent documentation pages. For example, we wouldn't have to describe where the files pane is every time we mention it.
... I think that there may be an argument for some repetition in description because a user may or may not have seen pages _other than the page they are on_ -- depending on how they find the D@R pages and how they have navigated within same. Or: there may be good reason to describe where the files pane is (nearly) very time (or at least, _many times_).

There are a number of things covered in this comment thread; gathering all in one place to get our paws around scope:
-  add an "intro to RStudio/the server" section where we outline what the different panels/tabs/windows are in RStudio, what they do, and where they are ... then, additional info about file paths, creating folders, making sure you know where you are, setting a working directory ...
-  Short description of the console, the environment pane, the files pane, etc., as well as any relevant tabs contained therein
- [Add a] section for importing data into the environment [including] a breakdown of the panels in the popup that comes up when you go to import data
- As an elaboration of R Markdown: creating code chunks, knitting, important chunk options e.g. eval = FALSE
- A description of the process of creating an R project
- [A description of] navigating between projects and also within a project
- [A description of the] difference between creating a new project and a new script

- maybe we could talk about doing GitHub version control?

And, a thought on structure:
- one page dedicated to just the different panels in RStudio UI + how to open or create new files ... then create other pages to address some of the other wonderful ideas you all have (flepaths, projects, rmarkdown feature explanations).

Looking at the above ideas, I believe one could group the new content into:
- rstudioServer: UI explanation, opening/creating new files within the server (?navigating filepaths within server?)
- rstudio projects (sharing w/in server, creating new, why do this v. creating a new script or RMD - the latter is relevant to server+desktop)
- rMarkdown features / deeper dive than what is on the website
- ?GitHub?

(re: GitHub: _I could swear  that we linked to HappyGitWithR linked from the doc pages_, and yet: I cannot find it at this moment. We could add a page on GitHub / R, which I would vote to be _very brief description_ and perhaps largely a _please go look at Jenny Bryan's excellent text_. Also, note that there is a mention of GitHub in [_other_ areas of the D@R site](https://www.reed.edu/data-at-reed/managing_data/version_control_backup.html) (add link?)

Process-wise: I would like to _start_ with the edits that we have to our current content, get those finalized + posted in the CMS, and _then_ we can talk about new structure. This could even be a topic for lunch next week, when we're all in the same room (!) in the mLab (!!) with Much Whiteboard.
